internal/db/rdb/mysql/mysql.go

```diff
@@ -251,23 +250,23 @@ func (m *mySQLClient) SetMeta(ctx context.Context, mv MetaVector) error {
 	}
 	defer tx.RollbackUnlessCommitted()
 
-	err = validateMeta(mv)
+	err = validateVector(mv)
 	if err != nil {
 		return err
 	}
 
-	_, err = tx.InsertBySql("INSERT INTO meta_vector(uuid, vector, meta) VALUES (?, ?, ?) ON DUPLICATE KEY UPDATE vector = ?, meta = ?",
+	_, err = tx.InsertBySql("INSERT INTO "+vectorTableName+"(uuid, vector, meta) VALUES (?, ?, ?) ON DUPLICATE KEY UPDATE vector = ?, meta = ?",
```

Comment:
```suggestion
	_, err = tx.InsertBySql("INSERT INTO "+vectorTableName+"(uuid, vector) VALUES (?, ?) ON DUPLICATE KEY UPDATE vector = ?",
```
and please remove `mv.GetVector()` on L261 and L262.